CHANGELOG.md

```diff
@@ -3,3 +3,6 @@
 * Feature: Add support for WebSEAL/IBM ISAM.
 * Breaking change: Connections to the LDAP server are no longer established
   automatically (see README).
+
+# 1.1.2
+* Feature: Handling Boolean Attributes
```

Comment:
I would reverse the order of the changelog (newer on top)

Reply:
I agree